buckifier/buckify_rocksdb.py

```diff
@@ -109,7 +109,8 @@ def generate_targets(repo_path):
         "rocksdb_test_lib",
         src_mk.get("MOCK_LIB_SOURCES", []) +
         src_mk.get("TEST_LIB_SOURCES", []) +
-        src_mk.get("EXP_LIB_SOURCES", []),
+        src_mk.get("EXP_LIB_SOURCES", []) +
+        src_mk.get("ANALYZER_LIB_SOURCES", []),
```

Comment:
Should this be in rocksdb_tools_lib, like in Makefile - https://github.com/facebook/rocksdb/blob/master/Makefile#L1048?

Reply:
done

Reply:
Looks like the analyzer lib is required by trace_analyzer_test. I'll add it back to rocksdb_test_lib